.github/workflows/avm.platform.sync-avm-modules-list.yml

```diff
@@ -0,0 +1,29 @@
+name: "avm.platform.sync-avm-modules-list"
+
+on:
+  schedule:
+    - cron: "30 4 * * *" # Every day at 4:30 am
+  workflow_dispatch:
+
+jobs:
+  sync-avm-modules-list:
+    runs-on: ubuntu-latest
+    steps:
+      - name: "Checkout"
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - env:
+          GH_TOKEN: ${{ github.token }}
+        name: sync avm modules list
+        shell: pwsh
+        run: |
+          # Load used functions
+          . (Join-Path $env:GITHUB_WORKSPACE 'avm' 'utilities' 'pipelines' 'platform' 'Get-AvmCsvData.ps1')
```

Comment:
Function does not seem to be used

Reply:
This may be - but it would be impossible to tell from the pipeline and even harder to understand later ;) Loading a functino that is used by a script should happen in that script and not outside. ;)

Reply:
Added in other comment.

Please note, you should then also pass the repo root in explicitely (just in case):

```PowerShell
$functionInput = @{
  Repo     = "${{ github.repository_owner }}/${{ github.event.repository.name }}"
  RepoRoot = $env:GITHUB_WORKSPACE
}
```

Reply:
done